packages/express/src/clerkClient.ts

```diff
@@ -0,0 +1,28 @@
+import type { ClerkClient } from '@clerk/backend';
+import { createClerkClient } from '@clerk/backend';
+
+import { loadApiEnv, loadClientEnv } from './utils';
+
+let clerkClientSingleton = {} as unknown as ClerkClient;
+
+export const clerkClient = new Proxy(clerkClientSingleton, {
+  get(_target, property: keyof ClerkClient) {
```

Comment:
Can you please remind me why we need the Proxy here? Is it because of the SK/PK errors we throw during initialization? 

I'd feel much more comfortable if we could remove the Proxy, no worries if its blocking though. However, we should figure out how we can avoid the need for workarounds like this

Reply:
We are using the Proxy to lazy load the envs when the `clerkClient` is actually used instead of loading the envs at the time of the `cerkClient` import.
Before we introduced the Proxy the customers using the `@clerk/clerk-sdk-node` had to import dotenv before importing the `@clerk/clerk-sdk-node`. Also due to autoformatting of files with eslint, the dotenv is moved after the clerk-sdk-node which was throwing errors of uninitialized pk/sk.

Reply:
We are using this pattern in the `@clerk/clerk-sdk-node`. We will discuss it in future refactorings.